### PR TITLE
Redirect NO and EN lessons

### PR DESCRIPTION
**Pull request by doomspork**
@ybur-yug this PR enables the changes you suggested awhile back RE: moving `en` lessons into a lesson code directory like the rest.

@nscyclone this should resolve the `no` language code.

Still testing this locally to make sure there aren't any issues.

### CODE REVIEW

**Comment by eksperimental** [comment]
I have found several issues, but I thin it will be better if I can submit a few commits. how can we do that @doomspork ?

File: en/lessons/advanced/otp-supervisors.md [hunk 1]
```diff
@@ -5,6 +5,8 @@ title: OTP Supervisors
 category: advanced
 order: 6
 lang: en
+redirect_from
```
Comment (eksperimental) [inline en/lessons/advanced/otp-supervisors.md]: you are missing a trailing `:`, this breaks the building

**Comment by doomspork** [comment]
@eksperimental if you'd like you can open a PR into this branch — would that do the trick?

**Comment by doomspork** [comment]
Closing as this was handled by @eksperimental in #1112

**Pull request closed without merging**